frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -77,9 +82,84 @@ pods:
         goal: FINISHED
         cmd: ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/nodetool cleanup -p {{TASKCFG_ALL_JMX_PORT}} -- $CASSANDRA_KEYSPACE_TABLES
         resource-set: sidecar-resources
+      backup-schema:
+        goal: FINISHED
+        cmd: ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/cqlsh --cqlversion="3.4.0" -e "DESC SCHEMA" node-0-server.{{SERVICE_NAME}}.mesos > container-path/schema.cql
```

Comment:
@nickbp it's not necessary, given that `cqlsh` will always be the version that shipped with the version of C* we're running against. Going to remove.

@mohitsoni it could use the VIP instead? Would that work?